src/game/game.hpp

```diff
@@ -732,6 +732,14 @@ class Game {
 	Title getTitleById(uint8_t id);
 	Title getTitleByName(const std::string &name);
 
+	const std::string &getSummaryKeyByType(uint8_t type) {
```

Comment:
You can move this to the cpp as well, this way you maintain the consistency of the declaration being in the cpp and the definition in the hpp